pkgs/applications/misc/klipperscreen/default.nix

```diff
@@ -1,28 +1,56 @@
-{ lib, stdenv, writeText, python3Packages, fetchFromGitHub, gtk3, gobject-introspection, gdk-pixbuf, wrapGAppsHook, librsvg }:
-python3Packages.buildPythonPackage rec {
+{ python3
+, fetchFromGitHub
+, wrapGAppsHook
+, gobject-introspection
+}:
+with python3.pkgs; buildPythonApplication rec {
   pname = "KlipperScreen";
   version = "0.3.2";
+  format = "other";
 
   src = fetchFromGitHub {
     owner = "jordanruthe";
-    repo = pname;
+    repo = "KlipperScreen";
     rev = "v${version}";
     hash = "sha256-LweO5EVWr3OxziHrjtQDdWyUBCVUJ17afkw7RCZWgcg=";
   };
-  patches = [ ./fix-paths.diff ];
 
-  buildInputs = [ gtk3 librsvg ];
-  nativeBuildInputs = [ wrapGAppsHook gdk-pixbuf gobject-introspection ];
+  nativeBuildInputs = [
+    gobject-introspection
+  ];
 
-  propagatedBuildInputs = with python3Packages; [ jinja2 netifaces requests websocket-client pycairo pygobject3 mpv six dbus-python numpy pycairo ];
+  buildInputs = [
+    wrapGAppsHook
+  ];
 
-  preBuild = ''
-    ln -s ${./setup.py} setup.py
+  pythonPath = [
```

Comment:
Is the latest recommendation to use `pythonPath` instead of `propagatedBuildInputs` for Python packaging?

I haven't been following development too closely.

Reply:
That's not a library, so I think it doesn't need to be propagated~